content/en/docs/_index.md

```diff
@@ -7,12 +7,12 @@ LinkTitle: "Documentation"
 
 section1:
   title: KubeSphere Documentation
-  content: Learn how to build and manage cloud native applications using KubeSphere Container Platform. Get documentation, example code, tutorials, and more.
+  content: Learn how to build and manage cloud-native applications using KubeSphere Container Platform. Get documentation, example code, tutorials, and more.
```

Comment:
`cloud native` is more common-used.

Reply:
I don't think there is a very obvious preference for whether the hyphen should be used. I changed it because I noticed some major cloud platform are using cloud-native. For example: Google and RedHat.
<img width="967" alt="截屏2020-09-02下午4 14 18" src="https://user-images.githubusercontent.com/65327072/91956692-a4144f00-ed37-11ea-86e1-5a7894aa23d5.png">
![image](https://user-images.githubusercontent.com/65327072/91956728-af677a80-ed37-11ea-8f6c-4fe6e9612b46.png)

Reply:
ok, actually both two are correct and common-used, I always see `cloud native` in some popular cloud provider websites.